src/RedHat.Container/RedHat.Container.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think you could use the NoTargets SDK here and skip a lot of the workarounds you had to do.

MSBuildSdks/src/NoTargets/README.md at main · microsoft ... - GitHub https://github.com/microsoft/MSBuildSdks/blob/main/src/NoTargets/README.md